pages/mesosphere/dcos/version-policy/index.md

```diff
@@ -83,98 +80,97 @@ Customers running DC/OS on non-supported platform components should upgrade to a
         <td><p style="text-align: center;">Docker CE 19.03.5<br>Docker CE 18.09.9</p></td>
         <td><p style="text-align: center;">Docker CE 19.03.5<br>Docker CE 18.09.9</p></td>
         <td><p style="text-align: center;">Docker CE 19.03.5<br>Docker CE 18.09.9</p></td>
     </tr>
 </table>
 
 ### <a name="Ubuntu-support-matrix"></a>Ubuntu support matrix
+*NOTE: Starting with DC/OS 2.0, we have discontinued support for Ubuntu Linux and have ceased qualifying it as an Operating System*.
 <table class="table">
     <tr>
     <th><strong>Platform Component</strong></th>
     <th><strong>DC/OS 2.0 Latest Stable</strong></th>
     <th><strong>DC/OS 1.13 Latest Stable</strong></th>
-    <th><strong>DC/OS 1.12 Latest Stable</strong></th>
     </tr>
     <tr>
         <td>Ubuntu 18.04.3 LTS</td>
         <td><p style="text-align: center;">Docker CE 19.03.5<br>Docker CE 18.09.9</p></td>
         <td><p style="text-align: center;">Docker CE 19.03.5<br>Docker CE 18.09.9</p></td>
-        <td><p style="text-align: center;">Docker CE 19.03.5<br>Docker CE 18.09.9</p></td>
     </tr>
     <tr>
         <td>Ubuntu 16.04.6 LTS</td>
         <td><p style="text-align: center;">Docker CE 19.03.5<br>Docker CE 18.09.9</p></td>
         <td><p style="text-align: center;">Docker CE 19.03.5<br>Docker CE 18.09.9</p></td>
-        <td><p style="text-align: center;">Docker CE 19.03.5<br>Docker CE 18.09.9</p></td>
     </tr>
 </table>
 
 ### <a name="Oracle-support-matrix"></a>Oracle support matrix
+*NOTE: Starting with DC/OS 2.0, we have discontinued support for Oracle Linux and have ceased qualifying it as an Operating System*.
 <table class="table">
     <tr>
     <th><strong>Platform Component</strong></th>
     <th><strong>DC/OS 1.13 Latest Stable</strong></th>
-    <th><strong>DC/OS 1.12 Latest Stable</strong></th>
     </tr>
     <tr>
         <td>Oracle Linux 7.6 </td>
         <td><p style="text-align: center;">Docker CE 18.09.2<br>Docker EE 18.09.2</p></td>
-        <td><p style="text-align: center;">Docker CE 18.09.2<br>Docker EE 18.09.2</p></td>
     </tr>
     <tr>
         <td>Oracle Linux 7.5 </td>
         <td><p style="text-align: center;">Docker CE 18.09.2<br>Docker EE 18.09.2</p></td>
-        <td><p style="text-align: center;">Docker CE 18.09.2<br>Docker EE 18.09.2</p></td>
     </tr>
     <tr>
         <td>Oracle Linux 7.4 </td>
         <td><p style="text-align: center;">Docker CE 18.09.2<br>Docker EE 18.09.2</p></td>
-        <td><p style="text-align: center;">Docker CE 18.09.2<br>Docker EE 18.09.2</p></td>
     </tr>
 </table>
 
 ### <a name="CoreOS-support-matrix"></a>CoreOS support matrix
-NOTE: Starting with DC/OS 2.0, we have discontinued support for CoreOS and have ceased qualifying it as an Operating System. This chart shows the final qualified versions of DC/OS for the final qualified versions CoreOS.
+As of May 26th 2020 CoreOS Container Linux has reached its [end of life](https://coreos.com/os/eol/) and will no longer receive updates.
```

Comment:
We should mention that we are recommending customers to use FlatCar Linux as an alternative?